app/views/books/show.html.erb

```diff
@@ -13,7 +13,23 @@
       <img  src="/images/progit2.png" width="118" height="157"/>
       <div style="light">2nd Edition (2014)</div>
       <div class="switch"><a href="/book/<%= @book.code %>/v1">Switch to 1st Edition</a></div>
-    </div>
+      <div class="ebook-download">
+        <h2>Download Ebook</h2>
+        <div class="ebooks">
+          <% if @book.ebook_pdf %>
+            <a href="<%= @book.ebook_pdf %>"><img width="50px" src="/images/pdf.png"/></a>
+          <% end %>
+          <% if @book.ebook_epub %>
```

Comment:
shouldn't this be contained in a div also?

Reply:
I just reinserted a code snippet that was deleted before. But at that time, 4 outputs were generated and the presentation was laid out in a 2x2 matrix. I kept the design but ditched the html download, which is useless.